src/utils/general.hpp

```diff
@@ -17,6 +17,11 @@
 #include <algorithm>
 #include <cctype>
 #include <functional>
+#include <string>
+
+#if defined(__clang__) || defined(__GNUG__)
+#include <cxxabi.h>
```

Comment:
Shouldn't this kind of thing be encapsulated in the corresponding cpp file instead?

Reply:
There isn't a `utils/general.cpp`, and I wasn't sure where else would be a better place for this kind of thing.

Reply:
If you're explicitly trying to avoid adding a new source file, I guess you could just drop it in wesnoth.cpp… but probably adding a new source file (whether general.cpp or something else) is the best option here.

Reply:
I assume I can create a `utils/general.cpp` without any issues.

Reply:
Done.